OracleSOABPM/dockerfiles/12.2.1.2-soabpm/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle BPM 12.2.1.2 Quick Start
+#
+# Base image of this dockerfile is the server JRE 8 docker image.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_12.2.1.2.0_bpm_quickstart_generic.jar & 
+#     fmw_12.2.1.2.0_bpm_quickstart_generic2.jar
+# (2) Download from OTN
+#     http://www.oracle.com/technetwork/middleware/soasuite/downloads/index.html
+#     - Download Queue/Selected Item
+#       - Oracle SOA Suite for Oracle Middleware (BPM, FMW, Service Oriented
+#         Architecture
+#       - Linux x86-64
+#       - CONTINUE
+#     - Download Queue/Select Release
+#       - Oracle SOA Suite for Oracle Middleware 12.2.1.2.0 for Linux x86-64
+#       - CONTINUE
+#     - Accept Terms
+#     - Download the following
+#       - V779125-01_1of2.zip
+#         Oracle Fusion Middleware 12c (12.2.1.2.0) Business Process Management
+#         Quick Start for Developers
+#       - V779125-01_2of2.zip
+#         Oracle Fusion Middleware 12c (12.2.1.2.0) Business Process Management
+#         Quick Start for Developers
+#       - Unzip to get the 2 jars mentioned above
+#
+# Pull base image
+# ---------------
+FROM container-registry.oracle.com/java/serverjre:8u131
```

Comment:
Customers are going to build this once and then distribute internally. The images we provide on OCR and other registries will be built off the FMW Infrastructure image. This needs to change to inherit from FMW Infrastructure, because that's exactly what it was designed for: to be the base for SOA/BPM/OSB.